webcompat/form.py

```diff
@@ -215,4 +215,7 @@ def build_formdata(form_object):
             image_url=form_object.get('image_upload').get('url'))
     # Append "from webcompat.com" message to bottom (for GitHub issue viewers)
     body += u'\n\n{0}'.format(GITHUB_HELP)
-    return {'title': summary, 'body': body}
+    rv = {'title': summary, 'body': body}
+    if 'label' in form_object:
+        rv.update({'labels': [form_object.get('label')]})
```

Comment:
Hmm, looking again at the Issues API docs,

> Labels to associate with this issue. NOTE: Only users with push access can set labels for new issues. Labels are silently dropped otherwise.

This probably means that if someone creates an issue authenticated as themselves, it will fail. webcompat-bot proxied issues would be OK. So maybe this is the wrong layer, and I should do this in the labeler webhook instead.